### ISSUE TITLE
bug: integration with rxjs 5.5.0-beta.0: import path gets appended with "/index"

### ISSUE THREAD

**Issue by deebloo** [dherges/ng-packagr#133]
## Type of Issue

- [X] Bug Report

## Description

Currently cannot create packages with rxjs 5.5.0-beta.0

#### How To Reproduce
```
npm i rxjs@5.5.0-beta.0
```

and then use one of the new lettable operator
```TS
import { from } from 'rxjs/observable/from';
import { map } from 'rxjs/operators';

export foo = from([0, 1, 2, 3, 4, 5]).pipe(map(x => x * 2));
```

#### Expected Behaviour

I would expect this to work the same as other rx versions. Instead I get the following error.

```
'rxjs/operators/index' is imported by src/app/common/.ng_build/ts/components/phone-number/phone-number.component.js, but could not be resolved – treating it as an external dependency
'rxjs/operators/index' is imported by src/app/common/.ng_build/ts/services/countries.service.js, but could not be resolved – treating it as an external dependency
'rxjs/operators/index' is imported by src/app/common/.ng_build/@livesafe/common.es5.js, but could not be resolved – treating it as an external dependency
No name was provided for external module 'rxjs/operators/index' in options.globals – guessing 'index'
```

If I try to add them to externals I get 

```
ERROR in ./src/app/common/dist/@livesafe/common.es5.js
Module not found: Error: Can't resolve 'rxjs/operators/index' in '/Users/dannyblue/Documents/projects/ng-livesafe/src/app/common/dist/@livesafe'
resolve 'rxjs/operators/index' in '/Users/dannyblue/Documents/projects/ng-livesafe/src/app/common/dist/@livesafe'
  Parsed request is a module
  using description file: /Users/dannyblue/Documents/projects/ng-livesafe/src/app/common/dist/package.json (relative path: ./@livesafe)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /Users/dannyblue/Documents/projects/ng-livesafe/src/app/common/dist/package.json (relative path: ./@livesafe)
    resolve as module
```
etc etc

#### Version Information

```
ng-packagr: v1.2.0
node: v8.3.0
@angular: v4.4.3
rxjs: v5.5.0-beta.0
zone.js: v0.8.17
```

**Comment by deebloo** [comment]
I can seem to find how to add these to externals.

**Comment by deebloo** [comment]
*can't

**Comment by deebloo** [comment]
@dherges Ive been digging but I can't figure out how to configure externals to make this work.
```JSON
{
    "externals": {
      // This fixes the first error but I can't figure out what to put as the value
      "rxjs/operators/index": "Rx.IDK.What.To.PutHere"
    }
}
```

**Comment by deebloo** [comment]
it seems to work if I import the operators directly. 
```
import { of } from 'rxjs/observable/of';
import { tap } from 'rxjs/operators/tap';
import { map } from 'rxjs/operators/map';
```
Is there something special to configure when the source is a barrel?

**Comment by dherges** [comment]
I tried to give [another explanation of lib.externals](https://github.com/dherges/ng-packagr/issues/135#issuecomment-332730342) and how I use them primarily.  Regarding the specific RxJS `rxjs/operators/map` import path: I don't know. Maybe try to figure out how the Angular core packages configure rollup as they also use this import syntax (afaik)

**Comment by dherges** [comment]
There is also an issues with `something/index` import paths ... when importing `something`, the path is rewritten to `import/index`, see #101. Is this the case?

**Comment by deebloo** [comment]
Yes it is rewritten to index

**Comment by dherges** [comment]
Hi @deebloo,

can you check / confirm that it's solved with angular 4.4.5?

See: https://github.com/dherges/ng-packagr/issues/101#issuecomment-337114529

**Comment by dherges** [comment]
If no, please comment here and/or re-open!
